app/build.gradle

```diff
@@ -90,20 +90,12 @@ android {
         staging {
             applicationId "com.nimbl3.staging"
         }
-
-        staging_internal {
-            applicationId "com.nimbl3.staging"
-            versionNameSuffix ".${project.versionNameSuffix}"
-        }
     }
 
     sourceSets {
         staging {
             res.srcDirs = ['src/staging/res']
         }
-        staging_internal {
-            res.srcDirs = ['src/staging/res']
-        }
```

Comment:
Should we do something about `staging_internal` inside the `data` submodule as well?